pkg/randchar/example_test.go

```diff
@@ -0,0 +1,42 @@
+package randchar
```

Comment:
Shall we change the package to the external test package `randchar_test`?

I think this will make the examples more representative for how the `randchar` package is consumed: from outside the package itself. The examples will then also import the `randchar` package and use the prefix (e.g. `randchar.Min` instead of just `Min`). Furthermore, we ensure the examples don't use unexported code.

Reply:
Good idea! We could also do this with the Client examples.